editor/components/post-saved-state/index.js

```diff
@@ -59,8 +60,8 @@ export function PostSavedState( { isNew, isPublished, isDirty, isSaving, isSavea
 
 	return (
 		<Button className={ classnames( className, 'button-link' ) } onClick={ onClick }>
```

Comment:
should we do `if ( isPublished ) { return null; }` instead?

Reply:
ok, unless we still want to use this component to show the autosave progress while in action? right now it says 'Autosaving' then 'Saved'. if we do remove this entirely, would disabling/re-enabling the update button be enough to indicate autosaving?

Reply:
Calling designers :P @jasmussen @karmatosed 

As I said, maybe we'd want to separate "autosaving" from "saving drafts" but I'll let our awesome designers find a solution for this. And I don't consider this a blocking for this PR.